### PR TITLE
Sync QueryAssistant, Recommendation PutFeedback APIs

### PR DESCRIPTION
**Pull request by hochengh**
*Issue #, if available:*

*Description of changes:*
This commit is to synchronize the QueryAssistant, GetRecommendations and PutFeedback APIs to latest version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# amazon-q-connectjs
```
Comment (Brunation11) [inline README.md:1]: meant to remove the space here?

File: README.md [hunk 2]
```diff
@@ -119,7 +120,7 @@ Initializing the QConnectJS client is the fist step to verify that you have ever
           instanceUrl: instanceUrl,                                        // REQUIRED
           endpoint: "https://my-instance-domain.awsapps.com/connect/api",  // optional, defaults to '<instanceUrl>'
           callSource: "agent-app",                                         // optional, defaults to 'agent-app'
-          serviceId: 'AmazonQConnect',                                     // optional, defaults to 'AmazonQConnect'
+          serviceId: 'AmazonQConnect',                                             // optional, defaults to 'AmazonQConnect'
```
Comment (Brunation11) [inline README.md:123]: spacing is off.

File: README.md [hunk 3]
```diff
@@ -260,52 +261,59 @@ Performs a manual search against the specified assistant. To retrieve recommenda
...
-        "title": {
-          "highlights": [
-            {
-              "beginOffsetInclusive": "number",
-              "endOffsetExclusive": "number"
+   "nextToken": "string",
+   "results": [
+      {
+         "data": {
+            "details": { ... },
```
Comment (Brunation11) [inline README.md:283]: any reason to show show the nested structure for these?

File: README.md [hunk 4]
```diff
@@ -401,47 +419,61 @@ Retrieves recommendations for the specified session. To avoid retrieving the sam
...
-        },
-        "title": {
-          "highlights": [
-            {
-              "beginOffsetInclusive": "number",
-              "endOffsetExclusive": "number"
+   "recommendations": [
+      {
+         "data": {
+            "details": { ... },
```
Comment (Brunation11) [inline README.md:433]: same here.

File: README.md [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
 
 QConnectJS supports the following APIs:
-* [QueryAssistant](#QueryAssistant)
-* [GetContact](#GetContact)
-* [GetRecommendations](#GetRecommendations)
-* [NotifyRecommendationsReceived](#NotifyRecommendationsReceived)
-* [GetContent](#GetContent)
-* [SearchSessions](#SearchSessions)
-* [ListIntegrationAssociations](#ListIntegrationAssociations)
+* [QueryAssistant](#queryassistant)
```
Comment (Brunation11) [inline README.md:8]: does the case change here matter?

File: src/bundle.ts [hunk 6]
```diff
@@ -8,11 +8,12 @@ import {
   GetRecommendations,
   NotifyRecommendationsReceived,
   GetContact,
+  PutFeedback,
 } from './index';
 
 (() => {
   const connect = (global as any).connect || {};
-  const qconnectjs = connect.qconnectjs || {};
+  const qconnectjs = connect.QConnectJS || {};
```
Comment (Brunation11) [inline src/bundle.ts:16]: this will always resolve to {} since right below you assign connect.qconnectjs in lowecase.

File: README.md (hunk 5 above)
Comment (hochengh) [reply README.md:8]: case does matter, need to be lower case so the anchor link can work

File: README.md (hunk 1 above)
Comment (hochengh) [reply README.md:1]: I don't think so, probably want it to be an anchor link. Will fix it!

File: src/bundle.ts (hunk 6 above)
Comment (hochengh) [reply src/bundle.ts:16]: let me fix it!

File: README.md (hunk 3 above)
Comment (hochengh) [reply README.md:283]: They are union type, so hard to display.

**Review by Brunation11 — APPROVED** [review]

**Pull request merged**